corelib-storage/src/main/java/eu/europeana/corelib/edm/utils/EdmUtils.java

```diff
@@ -203,7 +203,7 @@ private static void appendQualityAnnotations(RDF rdf,
                 QualityAnnotation qualityAnnotation = new QualityAnnotation();
                 resultList.add(qualityAnnotation);
 
-                if (EuropeanaUriUtils.isUri(anno.getAbout()) || preserveIdentifiers) {
```

Comment:
Are you sure about this change?
There was a reason for those checks(Which I don't remember unfortunately..).
I know that at least metis reprocessing tool is using the EdmUtils, hopefully that doesn't break any future use of the class.

Reply:
Could you point me to the repo where you are using EDMUtils, I can have a look if it will affect metis

Reply:
As I said it's in metis-tools/metis-reprocessing.
I think it's not used else where.
Not sure it matters where it's used, I guess it's related to the underlying data.
We are using the "preserveIdentifiers" true, so it shouldn't really affect the script as it is.
I was just wondering(and wanted to make sure there isn't any other case, and there wouldn't be in the future) where that check was used(perhaps it was when we did the first re-index using the europeana oai-pmh or something like that)

Reply:
Okay, so we use it only to generate RDF response with **preserveIdentifier** as false. I have verified the responses and the data for a few records and it doesn't affect us that way. All the about and resource fields of proxy, aggregatedCho, aggregation, Europeana Aggregation are checked for **preserveIdentifier**. The data is a relative uri that usually starts with '/' and we add a base URL depending upon that.

Reply:
Okay then!:)